### ISSUE TITLE
Intellisense not working for iterated variable inside for loop

### ISSUE THREAD

**Issue by aka-STInG** [HaxeFoundation/intellij-haxe#600]
E.g.
```
for (x in arr)
{
     x.someMethod() <- unresolved symbol
}
```

**Comment by EricBishton** [comment]
Using Vector<T> in OpenFL, right?  This bug does not appear when using Array<T>, like the following:

```haxe
package ;

class MinimalParseArray {
    public function doSomething() {
        var vec = new Array<String> ();

        trace(vec[0].length);

        for (vecelem in vec) {
            trace(vecelem.length);
            trace(vecelem.charAt(0));
        }
    }
}
```

Just to be clear, I am NOT advocating using Array<T> instead of Vector<T>.  I'm merely pointing out the context of the error.

**Comment by EricBishton** [comment]
Fixed in 0.11.0-RC3.
